vsm

```diff
@@ -22,6 +23,34 @@ LOGIC_REPLACE = {'||': 'or',
 
 KEYWORDS = ['condition', 'emit']
 
+start_time_ms = 0
+
+class Log(object):
+    '''
+        Utility class for logging messages
+    '''
+
+    '''
+        Log an informative (non-error) message
```

Comment:
I think these should go inside the function definition, so that they become docstrings (will be seen when you help(Log.i) in the python interpreter).